.github/workflows/codemagic.yaml

```
@@ -0,0 +1,15 @@
+workflows:
```

actionlint (1.7.7)

unexpected key "workflows" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)